app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,16 @@ def show
     @post = User.find_by(id: params['user_id']).posts.find_by(id: params['id'])
     @comment = Comment.where(post_id: @post)
   end
+
+  def new
+    @user = self
+  end
+
+  def create
+    @values = params[:post]
+    @post = Post.new(title: @values['title'], text: @values['text'], author: User.find_by(id: params['user_id']))
+    return unless @post.save
+
+    redirect_to "/users/#{params['user_id']}"
+  end
```

Comment:
# Optional:
I will suggest you add the params method. Open the [strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters) method here to learn more
- Example
```ruby
 private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```